client/src/utils/scrollbar-width.ts

```diff
@@ -1,6 +1,6 @@
 import store from 'store';
 
 export function getScrollbarWidth(): number {
-  const storedWidth = parseInt(store.get('monacoScrollbarWidth'));
+  const storedWidth = parseInt(store.get('monacoScrollbarWidth') as string);
```

Comment:
```suggestion
  const storedWidth = store.get('monacoScrollbarWidth') as number;
```
Turns out `get` uses `JSON.parse` under the hood, so `parseInt` isn't necessary.

Reply:
cool cool. fixed.